nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -79,7 +80,16 @@ def target_point(target_type: TargetType,
 
     @staticmethod
     def quantizer_insertion_command(target_point: ONNXTargetPoint,
-                                    parameters: QuantizerLayerParameters) -> ONNXQuantizerInsertionCommand:
+                                    quantizer_config: QuantizerConfig,
+                                    statistics: Union[MinMaxTensorStatistic, np.ndarray],
+                                    node: NNCFNode) -> ONNXQuantizerInsertionCommand:
+        if isinstance(statistics, MinMaxTensorStatistic):
+            axis = 1 if quantizer_config.per_channel else None
+            parameters = calculate_activation_quantizer_parameters(statistics, quantizer_config, axis)
+        else:
+            axis = node.metatype.weight_definitions.weight_channel_axis if quantizer_config.per_channel else None
```

Comment:
Could be `node.metatype.weight_definitions.weight_channel_axis` value None? Or what value should it have if it is MatMul